app/components/FAQ.tsx

```diff
@@ -1,3 +1,67 @@
 export const FAQ = () => {
-  return <div>FAQ</div>;
+  return (
+    <section>
+      <h1 className="mb-[4.5rem] text-center text-white text-[3.5rem] font-semibold">
```

Comment:
This should be an h2.

Reply:
Got it! No problem. I gonna refactor it 💯